src/docs/become-committer.md

```diff
@@ -2,6 +2,9 @@
 title: 'Becoming a committer'
 description: 'How does one become a V8 committer? This document explains.'
 ---
+
```

Comment:
Nit: we don't usually have this newline here.

Reply:
Ack, removed.